### PR TITLE
doc updates to reference mac os x support for source installs #647

### PR DESCRIPTION
**Pull request by agentpoyo**
## Description
MacOS source install verification and proof of concept. 

## Issues:
Refs: #647 

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [x] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

**Comment by agentpoyo** [comment]
@taylor #647 in Peer Review

**Review by taylor — APPROVED** [review]
@agentpoyo looks good. the one addition you could make is MacOS on Intel. I can not test Arm (yet)

**Comment by agentpoyo** [comment]
> @agentpoyo looks good. the one addition you could make is MacOS on Intel. I can not test Arm (yet)

We actually don't mention ARM for Linux or WSL, should we amend to support those (I think we've tested but can't recall).

**Comment by agentpoyo** [comment]
> @agentpoyo looks good. the one addition you could make is MacOS on Intel. I can not test Arm (yet)

@taylor We actually mention Intel (amd64) only supported for all platforms in the [FAQ](https://github.com/cncf/cnf-testsuite/blob/main/FAQ.md)

File: FAQ.md [hunk 1]
```diff
@@ -37,7 +37,7 @@
 <details> <summary>What platforms are supported by the CNF Test Suite?</summary>
 <p>
 
-- The CNF Test Suite runs on most major Linux distributions and WSL (Windows Subsystem for Linux).
+- The CNF Test Suite runs on most major Linux distributions, Mac OS X (source install only) and WSL (Windows Subsystem for Linux).
```
Comment (taylor) [inline FAQ.md:40]:
```suggestion
- The CNF Test Suite runs on most major Linux distributions, Mac OS X (Intel + source install only) and WSL (Windows Subsystem for Linux).
```

File: SOURCE_INSTALL.md [hunk 2]
```diff
@@ -65,7 +65,9 @@ cd tools/ && git clone https://github.com/crosscloudci/k8s-infra.git
 
 ### Installation
 
-We can assume you have access to a working kubernetes cluster. We recommend only running the cnf-testsuite on dev or test clusters.
+We can assume you have access to a working kubernetes cluster. We recommend only running the cnf-testsuite on dev or test clusters. The source install steps have been verified on most Linux distributions (Ubuntu, Debian and CentOS), Mac OS X and WSL as long as crystal-lang v0.35.1 is installed.
```
Comment (taylor) [inline SOURCE_INSTALL.md:68]:
```suggestion
We can assume you have access to a working kubernetes cluster. We recommend only running the cnf-testsuite on dev or test clusters. The source install steps have been verified on most Linux distributions (Ubuntu, Debian and CentOS), Mac OS X (Intel only) and WSL as long as crystal-lang v0.35.1 is installed.
```

**Review by taylor — CHANGES_REQUESTED** [review]
2 updates for intel-only for macos

File: SOURCE_INSTALL.md (hunk 2 above)
Comment (agentpoyo) [reply SOURCE_INSTALL.md:68]: Updated the location of the platforms to the TECH FAQ area before architecture support is mentioned.

**Review by taylor — APPROVED** [review]

**Pull request merged**